opensrp-chw/src/kk/java/org/smartregister/chw/actionhelper/CareGiverResponsivenessActionHelper.java

```diff
@@ -0,0 +1,86 @@
+package org.smartregister.chw.actionhelper;
+
+
+import android.content.Context;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.LocalDate;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.smartregister.chw.anc.actionhelper.HomeVisitActionHelper;
+import org.smartregister.chw.anc.domain.VisitDetail;
+import org.smartregister.chw.anc.model.BaseAncHomeVisitAction;
+import org.smartregister.chw.anc.util.JsonFormUtils;
+import org.smartregister.domain.Alert;
+import org.smartregister.immunization.domain.ServiceWrapper;
+
+import java.util.List;
+import java.util.Map;
+
+import timber.log.Timber;
+
+/**
+ * Created by Kassim Sheghembe on 2022-03-28
+ */
+public class CareGiverResponsivenessActionHelper extends HomeVisitActionHelper {
+    private String caregiver_interacts_with_child;
+    private String caregiver_comfort_child;
+    private String caregiver_response_cue;
+    private Alert alert;
+
+    public CareGiverResponsivenessActionHelper(Alert alert) {
+        this.alert = alert;
+    }
+
+    @Override
+    public void onJsonFormLoaded(String jsonString, Context context, Map<String, List<VisitDetail>> details) {
+        super.onJsonFormLoaded(jsonString, context, details);
+    }
+
+    @Override
+    public void onPayloadReceived(String jsonString) {
+        try {
+            JSONObject jsonObject = new JSONObject(jsonString);
+            JSONArray fields = JsonFormUtils.fields(jsonObject);
+
+            caregiver_interacts_with_child = JsonFormUtils.getFieldValue(fields, "caregiver_interacts_with_child");
+            caregiver_comfort_child = JsonFormUtils.getFieldValue(fields, "caregiver_comfort_child");
+            caregiver_response_cue = JsonFormUtils.getFieldValue(fields, "caregiver_response_cue");
+
+        } catch (JSONException e) {
+            Timber.e(e);
+        }
+
+    }
+
+
+
+    @Override
+    public String evaluateSubTitle() {
+        return null;
+    }
+
+    @Override
+    public BaseAncHomeVisitAction.Status evaluateStatusOnPayload() {
+        String CAREGIVER_INTERACTING_WITH_CHILD = "Moves toward and with child, and talks to or makes sounds with child, responds to child";
```

Comment:
Same thing for this one, how will this work when the language selected is swahili